packages/ui/src/theme/tokens/components/passwordField.ts

```diff
@@ -37,6 +50,54 @@ export const passwordfield: Required<PasswordFieldTokens<'default'>> = {
       },
       color: { value: '{components.button._disabled.color.value}' },
     },
+    _error: {
+      color: { value: '{components.button.outlined.error.color.value}' },
+      backgroundColor: {
+        value: '{components.button.outlined.error.backgroundColor.value}',
+      },
+      borderColor: {
+        value: '{components.button.outlined.error.backgroundColor.value}',
```

Comment:
```suggestion
        value: '{components.button.outlined.error.borderColor.value}',
```

Is this correct? Looks like there might be a mismatch between borderColor property using a backgroundColor value

Reply:
Good catch